content/en/docs/tutorials/stateful-application/elasticsearch.md

```diff
@@ -0,0 +1,380 @@
+---
+title: "Building a Search Engine on Kubernetes using Elasticsearch"
+content_type: tutorial
+weight: 60
+---
+
+<!-- overview -->
+
+{{< warning >}}
+This deployment is not suitable for production use cases, you need to benchmark or size your cluster according to your use case.
+{{< /warning >}}
+
+{{< note >}}
+The files provided in this tutorial are using GA Deployment APIs and are specific to kubernetes version 1.16 and 1.20. You can also try it on newer Kubernetes version but some APIs might throw deprecated warnings. 
+{{< /note >}}
+
+## {{% heading "objectives" %}}
+
+1. You will learn about deploying a stateful application Elasticsearch and a visualizer like Kibana on Kubernetes.
+2. Then, you will build a simple React-based search app using Kubernetes as a backend.
+
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+The example shown on this page works with `kubectl` 1.16 and above.
+
+1. Brief understanding of REST APIs.
+2. High-level understanding of Elasticsearch.
+
+<!-- lessoncontent --> 
+
+# Creating an Elasticsearch Cluster
+
+There are many ways to run software like Elasticsearch on Kubernetes. For example, you could use the docker images or a helm chart. I’m using the official Kubernetes operator here. 
```

Comment:
There's no official Kubernetes operator - I'm concerned that this wording might suggest that the operator is endorsed by Kubernetes. What I think you're describing is the official Elasticsearch operator (for deploying onto Kubernetes) from Elastic.